docs/reference/migration/migrate_7_6.asciidoc

```diff
@@ -14,12 +14,44 @@ See also <<release-highlights>> and <<es-release-notes>>.
 //Installation and Upgrade Guide
 
 //tag::notable-breaking-changes[]
+[discrete]
+[[breaking_76_security_changes]]
+=== Security changes
+
+[discrete]
+==== {es} authentication API key privileges
```

Comment:
The word "authentication" does not seem to be necessary here. Also privileges are in the scope of authorization.